ee/clickhouse/views/actions.py

```diff
@@ -118,6 +103,19 @@ def people(self, request: Request, *args: Any, **kwargs: Any) -> Response:
             }
         )
 
+    @action(methods=["GET"], detail=True)
+    def count(self, request: Request, **kwargs) -> Response:
+        action = self.get_object()
+        query, params = format_action_filter(action)
+        if query == "":
+            return Response({"count": 0})
+
+        results = sync_execute(
```

Comment:
don't we want to keep the try/except block here too (seems better to show 0 & capture the exception for us than to error to the user)?

Reply:
There doesn't seem to be a reason for the try/catch here - when I looked through sentry there were no reported errors. No reason to be defensive then.